frontend/src/components/loading/spinner.svelte

```diff
@@ -10,7 +10,7 @@
 </script>
 
 <div class="flex justify-center items-center">
-  <div class="animate-spin rounded-full h-32 w-32 border-b-8 border-neutral" />
+  <div class="animate-spin rounded-full h-32 w-32 border-b-8 border-primary" />
```

Comment:
Not sure if primary should be used for everything.
Maybe a `state` color makes sense for this
![image](https://user-images.githubusercontent.com/43907402/151173117-c1d6252c-e67b-4681-8e72-5d948461bcc6.png)
(with luxury)

Reply:
In this specific case i feel like `primary` is what matches all the themes. 

![image](https://user-images.githubusercontent.com/50198099/151174247-53e11b33-514c-47f3-8360-eb0ea722f823.png)

![image](https://user-images.githubusercontent.com/50198099/151174290-296d0edd-d2f8-4aaa-8522-e22271738f60.png)

![image](https://user-images.githubusercontent.com/50198099/151174335-bda42013-0bd0-444e-909b-6861acf6036c.png)

![image](https://user-images.githubusercontent.com/50198099/151174361-b5e28329-173f-41f2-a574-7411ec4e5852.png)

![image](https://user-images.githubusercontent.com/50198099/151174386-b14449ca-0288-40a8-bd62-ac4774b53e82.png)

I have tried a lot of combinations, but this is what looks best IMO.

Reply:
True, it does looks really nice... I'll find somewhere else to use colors for hints/warnings/errors - Maybe the error page? :D

Reply:
That could be a good idea 👀